README.md

```diff
@@ -1 +1,141 @@
+
+To get a local copy up and running, follow these steps.
+
+- First you need to have an IDE installed in your PC. You also need to have a github account in order to clone the code to your local repository
+- Login to your github account and search for 'prg-04/Hello_Microverse'
+- Click the green 'code' button and copy the HTTPS url.
+- Open the your preferred IDE and open the terminal window. Write the following commands in the terminal 'git clone' followed by the url link you copied and press Enter on your keyboard. Git will start cloning the repository into your selected folder.
+- After cloning the repository change the directory using 'cd Hello_Microverse' to access the files.
+
+### Prerequisites
+
+In order to run this project you need:
+
+    * Visual Studio Code,Atom or your preferred IDE
+    * MAC, Linux, Windows Vista or newer -GitHub account
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open the your preferred IDE and open the terminal window. Write the following commands in the terminal 'git clone' followed by the url link you copied and press Enter on your keyboard. Git will start cloning the repository into your selected folder.
+- After cloning the repository change the directory using 'cd Hello_Microverse' to access the files.
+
+### Install
+
+Install this project with:
+
+- In your terminal window type: git clone https://github.com/prg-04/Hello_Microverse.git
+
+### Usage
+
+To run the project, execute the following command:
+
+- code Hello_Microverse .
+
+### Deployment
+
+You can deploy this project using:
+
+- A live server in your preferred IDE
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/prg-04)
+- Twitter: [@twitterhandle](https://twitter.com/Evanso-044)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Evanson-Karanja)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Carosel]**
+- [ ] **[Animations]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/prg-04/Hello_Microverse/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse Foundation for their amazing templates.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE/MIT.md) licensed.
```

Comment:
Good job with README. It is customized very well, indeed. 👍  But let's add a software license to our project to have a professional project. You can check the instructions below in the template's license section: https://github.com/microverseinc/readme-template/blob/master/README.md#-license-